internal/cmd/run/run.go

```diff
@@ -239,6 +240,13 @@ depends on the build system configured for the project.
 				return cmdutils.WrapSilentError(err)
 			}
 
+			// check if the fuzz test is a method of a class
+			if strings.Contains(args[0], "::") {
+				fuzzTest := strings.Split(args[0], "::")[0]
+				opts.targetMethod = strings.Split(args[0], "::")[1]
```

Comment:
```suggestion
				split := strings.Split(args[0], "::")
				fuzzTest, testMethod  := split[0], split[1]
```
I think this would be more readable and avoids the second time splitting the string